### PR TITLE
usecase of the command bin/magento dev:source-theme:deploy

### PR DESCRIPTION
**Pull request by Santoshziffity**
## Purpose of this pull request

This pull request (PR) of the use case of the command bin/Magento dev:source-theme: deploy

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.html

### CODE REVIEW

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md [hunk 1]
```diff
@@ -80,3 +80,13 @@ To create LESS files for the adminhtml, enter the following command:
 ```bash
 bin/magento dev:source-theme:deploy --locale="en_US" --area="adminhtml" --theme="Magento/backend" css/styles css/styles-old
 ```
+
+## Common issue faced by Developer when working
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:84]:
```suggestion
## Common issues faced by developers when working
```

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md [hunk 2]
```diff
@@ -80,3 +80,13 @@ To create LESS files for the adminhtml, enter the following command:
...
+
+In the development stage, when you are working with css, Js, knockout js and knockout template, the developer faces below issues
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:86]:
```suggestion
In the development stage, when you are working with CSS, JS, Knockout JS and Knockout template, the developer may face the following issues:
```

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md [hunk 3]
```diff
@@ -80,3 +80,13 @@ To create LESS files for the adminhtml, enter the following command:
...
+
+-  The changes in css , knockout js and knockout templates are not affecting on frontend when page load.
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:88]:
```suggestion
-  The changes in CSS , Knockout JS and Knockout templates are not reflected on frontend when page loads.
```

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md [hunk 4]
```diff
@@ -80,3 +80,13 @@ To create LESS files for the adminhtml, enter the following command:
...
+
+-  The developer has to delete the pub static folder or run static content deploy command every time when changes in js and knockout template file.
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:90]:
```suggestion
-  The developer has to delete the `pub/static` folder or run static content deploy command every time when there are changes in JS and Knockout template files.
```

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md [hunk 5]
```diff
@@ -80,3 +80,13 @@ To create LESS files for the adminhtml, enter the following command:
...
+
+If you are facing this kind of issue in development then create symlinks
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:92]: This is not a very helpful statement. Please expand on how this should be done specifically when the above issues are encountered.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
Hmm...I am not sure about this addition. Are these tips related to creating symlinks to LESS files? This seems more like general implementation tips.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
@Santoshziffity ?

**Comment by dobooth** [comment]
Closing as abandoned.

**Pull request closed without merging**